### ISSUE TITLE
[BUG] Code does not transpile

### ISSUE THREAD

**Issue by kpalatzky** [acrolinx/sidebar-sdk-js#10]
The current version of code does not transpile.

Error while TSC:

```bash
> @acrolinx/sidebar-sdk@0.4.46 tsc C:\workspace\github\sidebar-sdk-js
> tsc

src/message-adapter/message-adapter.ts:59:5 - error TS2322: Type '{ init: (...args: any[]) => void; configure: (...args: any[]) => void; checkGlobal: () => { checkId: string; }; onGlobalCheckRejected: (...args: any[]) => void; invalidateRanges: (...args: any[]) => void; onVisibleRangesChanged: (...args: any[]) => void; dispose: (...args: any[]) => void; }' is not assignable to type 'AcrolinxSidebar'.
  Object literal may only specify known properties, and 'dispose' does not exist in type 'AcrolinxSidebar'.

59     dispose: _.noop
       ~~~~~~~~~~~~~~~

test/spec/async-adapter.ts:140:7 - error TS2322: Type '{ init(_initParameters: InitParameters): void; configure(_config: SidebarConfiguration): void; checkGlobal(documentContent: string, _options: CheckOptions): Check; onGlobalCheckRejected(): void; invalidateRanges(invalidCheckedDocumentRanges: CheckedDocumentPart[]): void; onVisibleRangesChanged(_checkedDocumentRanges...' is not assignable to type 'AcrolinxSidebar'.
  Object literal may only specify known properties, and 'dispose' does not exist in type 'AcrolinxSidebar'.

140       dispose(_callback: () => void) {
          ~~~~~~~

test/spec/multi-plugin.ts:150:7 - error TS2322: Type '{ init(_initParameters: InitParameters): void; configure(config: SidebarConfiguration): void; checkGlobal(documentContent: string, _options: CheckOptions): Check; onGlobalCheckRejected(): void; invalidateRanges(invalidCheckedDocumentRanges: CheckedDocumentPart[]): void; onVisibleRangesChanged(_checkedDocumentRanges:...' is not assignable to type 'AcrolinxSidebar'.
  Object literal may only specify known properties, and 'dispose' does not exist in type 'AcrolinxSidebar'.

150       dispose(_callback: () => void) {
          ~~~~~~~


Found 3 errors.
```

I had a look at the Code and the current Version of `acrolinx/sidebar-interface` does not support `AcrolinxSidebar.dispose` (See error above). After removing all `dispose` the code transpiles again.

Tested with:

OS: Win 10 Pro
Node: 8.12.0
NPM: 6.4.1

**Comment by acromarco** [comment]
@kpalatzky Thank you for your bug report. 
We’ve overseen this problem, because our test pipeline uses the exact version from package-lock.json (using "npm ci").
Probably we should test it also automatically using "npm i".
I’ve fixed the code and will also prepare a new release, updating some dependencies by the way.

**Comment by kpalatzky** [comment]
@acromarco When will the release happen? Because later this day i will make a PR for a little change in the XeditorAdapter (We need this change after we released Xeditor 6.0). Would be nice if this change could be in there too!

**Comment by acromarco** [comment]
@kpalatzky OK, I will do the release after merging your PR, maybe tomorrow.
